### PR TITLE
Completed exercise npm-intro. - Jesus Araujo

### PR DESCRIPTION
**Pull request by j-alejandro-araujo**
Screenshot 2023-06-20 at 9 26 47 PM

### CODE REVIEW

File: npm-intro/npm-intro-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+- How can you create a `package.json` with `npm`?
+
+- What is a dependency and how do you add one to a package?
+
+- What happens when you add a dependency to a package with `npm`?
+
+- What is a devDependency and how do you add one to a package?
+
+- How do you define and run `npm` scripts? Why are these useful?
```
Comment (shawnkost) [inline npm-intro/npm-intro-quiz-notes.md:21]: Don't forget the quiz questions!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Need to answer the quiz questions!

File: npm-intro/npm-intro-quiz-notes.md (hunk 1 above)
Comment (j-alejandro-araujo) [reply npm-intro/npm-intro-quiz-notes.md]: Sorry about that, all good now!

Screenshot 2023-06-21 at 10 38 42 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**